learners-guild/start.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 echo "Loading Learners Guild Environment"
+
+cd "$ROOT"
+
 source .env
 for var in `cat .env | grep = | cut -f1 -d=`
 do
```

Comment:
Can we add a `cd -` at the end so that, when the script exits, the user is left where they were originally?